### ISSUE TITLE
show the amount and timestamp when the price was last updated

### ISSUE THREAD

**Issue by basc** [woheller69/spritpreise#25]
I really like Spritpreise app.

It would be helpful to show when the price was last updated like some other apps do.

example:
Diesel: € 1,569

> _**-3 ct vor 29 Minuten**_

In the API there is lastChange with timestamp and amount.

Thank You.

**Comment by woheller69** [comment]
Where in the API is it?

**Comment by basc** [comment]
https://creativecommons.tankerkoenig.de/swagger/#/search/get_stations_postalcode

fuels - lastChange

Am 20. April 2023 22:38:08 MESZ schrieb woheller69 ***@***.***>:
>Where in the API is it?
>

**Comment by woheller69** [comment]
Interesting. This is a different API. 

https://creativecommons.tankerkoenig.de/api/v4/stations/search/?lat=52.521&lng=13.438&rad=1.5&sort=dist&type=all&apikey=xxx

It is not available in the official API documentation at

https://creativecommons.tankerkoenig.de/  -> Info

which defines this API call:

https://creativecommons.tankerkoenig.de/json/list.php?lat=52.521&lng=13.438&rad=1.5&sort=dist&type=all&apikey=xxx

Will think about it when they update their official documentation. But it is a lot of work as the database also has to be modified.
And does it really help if you see that you just missed a good price by 5 minutes?

**Comment by martin-braun** [comment]
> And does it really help if you see that you just missed a good price by 5 minutes?

That's true, but your chances increase to catch the good price when it was changed recently.

Is there an update on this?

**Comment by woheller69** [comment]
The official API documentation is still the same. I have no plans to implement the other API just for this purpose

**Comment by bomm** [comment]
> And does it really help if you see that you just missed a good price by 5 minutes?

It does not help to see **what** price I just missed, but I agree with @martin-braun that the time of the last price update would be useful.

After installing Spritpreise today, I compared it with Benzinpreis-Blitz and PACE Drive. A few minutes ago, Spritpreise showed a price different from the other two. As Spritpreise shows the time of the last request I suspect that tankerkoenig might have missed or delayed an update of the price.
Benzinpreis-Blitz shows when the price was last updated. If this would be available in all apps I could find out which information is more recent.

Bodo
